server/routes/search.test.ts

```diff
@@ -221,13 +221,13 @@ describe('Prisoner search page', () => {
 
     describe('POST /search/prisoner', () => {
       it('should redirect to search results page with search query when no search term entered', () => {
-        return request(app).post('/search/prisoner').expect(302).expect('location', '/search/prisoner/results')
+        return request(app).post('/search/prisoner').send({}).expect(302).expect('location', '/search/prisoner/results')
```

Comment:
Quite a few tests failed on upgrade where they were testing no data being sent by just not calling `send()`. Fixed by adding `.send({})` which is more realistic as `req.body` comes through as `{}` on `POST` (but in other cases can be `undefined`).

Reply:
I think `.send` with JSON data needs changing to `.json`